build/db-schemas/000002_inverted_index.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE identification_service_areas ADD COLUMN IF NOT EXISTS cells INT64[];
```

Comment:
The advantage to this is that we will not risk dropping any data if a single step is corrupted. Each step is idempotent can can be run multiple times if needed.

Reply:
If 000002, 000003, and 000004 steps were combined into a single step, would that combined step have a risk of dropping data if the combined step were corrupted?  Would the combined step be idempotent?

Reply:
we can make it idempotent but it would be harder to debug if anything goes wrong, breaking down in small steps where 1 step adds columns, 1 step migrates the data, and 1 final step dropping the old data makes it really clean.